.pipelines/ci/templates/build-powertoys-ci.yml

```diff
@@ -1,15 +1,15 @@
 parameters:
   configuration: 'Release'
   platform: ''
```

Comment:
@rainersigwald does it matter if the incoming global property value is lower case?

https://github.com/microsoft/PowerToys/blob/a8d2aeedf21a92f9137421c3f5c4149970e7e4ee/.pipelines/ci/ci.yml#L29

Reply:
For a solution, not really, for confusing reasons! We parse the solution to find the configuration for each project, and pass those explicitly. In the solution metaproject that manifests as:

```xml
    <ProjectReference Include="C:\a\_work\1\s\src\common\logger\logger.vcxproj.metaproj">
      <AdditionalProperties>Configuration=Release; Platform=arm64</AdditionalProperties>
      <Platform>ARM64</Platform>
      <Configuration>Release</Configuration>
      <ToolsVersion>Current</ToolsVersion>
      <SkipNonexistentProjects>Build</SkipNonexistentProjects>
    </ProjectReference>
```

Where the casing there is determined by what's in the solution (fixed in this PR), not what's actually passed in, which is only used to case-insensitively find the solution-level platform.

Reply:
So the fact that the `AdditionalProperties` metadata has lowercase `arm64` and the `Platform` metadata has uppercase `ARM64` won't matter?  And if you didn't specify any `Platform` global property, MSBuild would use `ARM64` everywhere?

Reply:
Lol naturally I picked a bad example that was one of the ones that got fixed in the change in this PR. Should look more like

```xml
    <ProjectReference Include="C:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.PowerToys.Run.Plugin.Service\Microsoft.PowerToys.Run.Plugin.Service.csproj">
      <AdditionalProperties>Configuration=Release; Platform=ARM64</AdditionalProperties>
      <Platform>ARM64</Platform>
      <Configuration>Release</Configuration>
      <ToolsVersion>
      </ToolsVersion>
      <SkipNonexistentProjects>False</SkipNonexistentProjects>
    </ProjectReference>
```

(both upper case).

If you don't specify Platform for the solution as a global property, there's a default in the .sln and we pick that pretty much exactly as though you explicitly specified it.

Reply:
Ah okay good to know, thanks for clearing that up!